src/main/java/org/broadinstitute/hellbender/tools/spark/validation/CompareDuplicatesSpark.java

```diff
@@ -106,7 +105,7 @@ protected void runTool(final JavaSparkContext ctx) {
         }
         if (printSummary) {
             MatchType[] values = MatchType.values();
-            HashSet<MatchType> matchTypes = Sets.newHashSet(values);
+            Set<MatchType> matchTypes = Sets.newHashSet(values);
```

Comment:
good catch. No, it does not produce a linkedhashset. I changed it in a few places where appropriate